frontend/src/libs/components/app/styles.module.scss

```diff
@@ -0,0 +1,4 @@
+.container {
+  width: 100%;
+  min-height: 100vh;
```

Comment:
@andanf-e, @nickolay-grechkin  May be replace vhs with flex-grow: 1?

Reply:
Let's try 🙏

Reply:
done - [9909c07](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/9909c074f0b39edb6031a295c34fe404cf6730b2)